src/joyride/sci.cljs

```diff
@@ -46,10 +69,27 @@
                            (symbol? namespace)
                            (source-script-by-ns namespace)
                            (string? namespace) ;; node built-in or npm library
-                           (if (= "vscode" namespace)
+                           (cond
+                             (= "vscode" namespace)
                              (do (sci/add-class! @!ctx 'vscode vscode)
                                  (sci/add-import! @!ctx (symbol (str @sci/ns)) 'vscode (:as opts))
                                  {:handled true})
+
+                             (active-extension? namespace)
+                             (let [[module-name module] (extension-module namespace)
+                                   ns-sym (symbol (str @sci/ns))
+                                   refer (:refer opts)]
+                               (sci/add-class! @!ctx (symbol namespace) module)
+                               (sci/add-import! @!ctx ns-sym (symbol namespace) (:as opts))
+                               (when refer
+                                 (doseq [sym refer]
+                                   (let [prop (gobject/get module sym)
+                                         sub-sym (symbol (str ns-sym "$" module-name "$" sym))]
```

Comment:
Same here, use `(munge (symbol ...))`

Reply:
Yes, I think this should be the libname we are requiring, since this "sub" thing will be registered as a global class and then imported in the current ns.